lib/pkgcloud/openstack/client.js

```diff
@@ -189,7 +189,15 @@ Client.prototype._request = function (options, callback) {
     self.auth(function (err) {
       if (err) {
         self.emit('log::error', 'Error with inline authentication', err);
-        return errs.handle(err, callback);
+        if(callback){
+          return errs.handle(err, callback);
+        } else {
```

Comment:
remove the `else` clause here. Since we do `return errs.handle(err, callback)` we dont have to wrap it in an `else` which helps us to avoid unnecesary nesting.